### PR TITLE
Update locales files

### PR DESCRIPTION
**Pull request by WakeupTsai**

### CODE REVIEW

File: src/locales/en-US/network.js [hunk 1]
```diff
@@ -28,5 +24,5 @@ export default {
   'network.hint.uniqueName': '{name} has been used! Try a new name.',
   'network.hint.selectNode': 'Select a node',
   'network.hint.selectInterfaces': 'Select interfaces',
-  'network.hint.VLANTag': 'Please enter a number between 0 ~ 4095.',
-}
+  'network.hint.vlanTags': 'Please enter a number between 0 ~ 4095.',
```
Comment (sufuf3) [inline src/locales/en-US/network.js]: The whole name of VLAN is Virtual Local Area Network.
Not sure that convert VLAN to lower case is good?
What do you think?
Thanks.

File: src/locales/en-US/service.js [hunk 2]
```diff
@@ -1,11 +1,8 @@
 export default {
-  'service.name': 'name',
-  'service.namespace': 'namespace',
   'service.type': 'type',
   'service.selectors': 'selectors',
   'service.ports': 'ports',
-  'service.add': 'Add Service',
-  'service.createdAt': 'created at',
+  'service.add': 'add Service',
```
Comment (sufuf3) [inline src/locales/en-US/service.js]: `Add` is verb. Maybe need to update to `Add Service`?
How do you like?

File: src/locales/en-US/volume.js [hunk 3]
```diff
@@ -1,7 +1,6 @@
 export default {
-  'volume.name': 'name',
-  'volume.storageName': 'storage name',
-  'volume.accessMode': 'access mode',
+  'volume.storageName': 'storage Name',
+  'volume.accessMode': 'access Mode',
```
Comment (sufuf3) [inline src/locales/en-US/volume.js]: Let's use lowercase, according to this comment.
https://github.com/linkernetworks/vortex-portal/pull/33#issuecomment-418613890

File: src/locales/zh-Hant/auth.js [hunk 4]
```diff
@@ -9,20 +9,20 @@ export default {
...
-  'auth.alreadyHaveAnAccount': 'Already have an account?',
+  'auth.alreadyHaveAnAccount': '已有帳號?',
   'auth.hint.forget': '忘記密碼?',
-  'auth.hint.doNotHaveAnAccount': 'don\'t have a account?',
-  'auth.hint.reason.wrongUsernameAndPassword': 'Your username and password are wrong. Please try again or contact your admin.',
+  'auth.hint.doNotHaveAnAccount': '還沒有帳號?',
+  'auth.hint.reason.wrongUsernameAndPassword': '帳號密碼錯誤，請重新嘗試或者聯絡管理人員。',
   'auth.hint.signup.success': '註冊成功',
-  'auth.hint.signup.success.description': 'Please sign in with your email and password',
+  'auth.hint.signup.success.description': '請使你的帳號密碼來登入',
```
Comment (sufuf3) [inline src/locales/zh-Hant/auth.js]: 請使 **用** 你的帳號密碼來登入

File: src/locales/zh-Hant/auth.js [hunk 5]
```diff
@@ -9,20 +9,20 @@ export default {
...
   'auth.hint.signup.failure': '註冊失敗',
-  'auth.hint.signin.failure': 'Failed to Signin',
-  'auth.help.password': 'This is one-time password, please send to your user.'
+  'auth.hint.signin.failure': '登入失敗',
+  'auth.help.password': '此為一次性的密碼，請交給使用者。'
```
Comment (sufuf3) [inline src/locales/zh-Hant/auth.js]: 此為一次性的密碼，請 **提供** 給使用者。

File: src/locales/zh-Hant/hub.js [hunk 6]
```diff
@@ -1,6 +1,6 @@
 export default {
   'hub.imageHub': '映像倉庫',
-  'hub.login': 'Login to unlock private image registry',
+  'hub.login': '登入以進入私有 Image 庫',
```
Comment (sufuf3) [inline src/locales/zh-Hant/hub.js]: image registry = 映像庫
Ref: https://docs.microsoft.com/zh-tw/dotnet/standard/microservices-architecture/container-docker-introduction/docker-containers-images-registries

**Review by sufuf3 — COMMENTED** [review]
Tag 和 Label 都翻成 **標籤**
這樣要怎麼區別呀?

File: src/locales/en-US/network.js (hunk 1 above)
Comment (LucienLee) [reply src/locales/en-US/network.js]: keep `VLANTags` and I have to put the missed s back to source code.

**Comment by LucienLee** [comment]
Tag 和 Label 要看上下文，我有點不確定我們這邊的 Tag 跟 Label 是用在哪裡？

File: src/locales/zh-Hant/hub.js (hunk 6 above)
Comment (LucienLee) [reply src/locales/zh-Hant/hub.js]: okay

**Comment by hwchiu** [comment]
1.映像檔標籤
2.容器標籤

File: src/locales/en-US/volume.js (hunk 3 above)
Comment (LucienLee) [reply src/locales/en-US/volume.js]: keep as `access mode`,  `storage name`

**Comment by WakeupTsai** [comment]
vLAN tag的tag呢？

**Pull request merged**